models/invitation.ts

```diff
@@ -29,6 +30,8 @@ export const createInvitation = async (param: {
 }) => {
   const { teamId, invitedBy, email, role } = param;
 
+  createEvent("invitation.created", "Invitation created");
```

Comment:
Each event should send a payload to the webhook. For example, `invitation.created` event should send the `invitation` object as a payload. You can send the `payload` to the `svix.message.create` method.